osler/inventory/views.py

```diff
@@ -20,10 +24,15 @@ def get_queryset(self):
                     select_related('unit').\
                     select_related('category').\
                     select_related('manufacturer').\
-                    order_by('category', 'name').\
+                    order_by('category', 'name', 'dose', 'expiration_date').\
                     exclude(stock=0).all()
         return druglist
 
+    def get_context_data(self, **kwargs):
+        context = super(DrugListView, self).get_context_data(**kwargs)
+        context['patients'] = Patient.objects.all().order_by('last_name').select_related('gender')
```

Comment:
I imagine that this list of patients would get crazy long at some point... depending on how UMKC uses the function, could be better to also filter by active or not (whether they are in clinic that day). I imagine they won't be dispensing drugs to inactive patients?

Reply:
Ohhh good point! Let me ask Eshwar what he wants!